### DISCUSSION TITLE
Do you want Cyberbrain to be advertised yet?

### DISCUSSION THREAD

**Discussion by alexmojaki** [laike9m/Cyberbrain#78] [Q&A] [answered]
Someone posted about it on reddit: https://www.reddit.com/r/Python/comments/k3xy1b/laike9mcyberbrain_cyberbrain_python_debugging/

I seem to remember seeing something written somewhere about not advertising until you were ready to launch, but I can't find it. Am I remembering correctly? Was it removed?

**Comment by laike9m** [comment, marked as answer]
I removed it a few days ago cause I feel it might confuse people, also because it's ready for the "announcement" stage.

Besides the link you posted, I actually did it myself:
https://www.reddit.com/r/Python/comments/k32p48/cyberbrain_python_debugging_redefined/
